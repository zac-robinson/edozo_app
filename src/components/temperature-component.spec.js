import React from 'react';
import { shallow } from 'enzyme';

import Temperature from './temperature-component';

describe('Temperature component', () => {
  it('should render correctly', () => {
    const temperatureProp = {
      temp: '15',
      temp_max: '45555',
      temp_min: '-1222'
    };
    const component = shallow(<Temperature temperature={temperatureProp} />);

    expect(component).toMatchSnapshot();
  });
});
