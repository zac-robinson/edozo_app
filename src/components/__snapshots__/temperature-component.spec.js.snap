// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Temperature component should render correctly 1`] = `
<styled.div>
  High: 
  45555
  <styled.h1>
    15
  </styled.h1>
  Low: 
  -1222
</styled.div>
`;
